packages/core/src/cli/init.ts

```diff
@@ -0,0 +1,19 @@
+import { loadConfig } from '../config';
+import type { RslibConfig } from '../types';
+import { getAbsolutePath } from '../utils/helper';
+import type { CommonOptions } from './commands';
+
+export async function getRslibConfig(
+  options: CommonOptions,
+): Promise<{ root: string; rslibConfig: RslibConfig }> {
+  const cwd = process.cwd();
+  const root = options.root ? getAbsolutePath(cwd, options.root) : cwd;
```

Comment:
IIUC, the path of config file does not take account into the `root` calculation anymore?

Reply:
Yes, the `root` is as the base dir to find to config files.